sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-components";
+
+export const ContentPlacementContainerStyles = css`
+    :host {
+        contain: content;
```

Comment:
This looks like a block element - can you use the `display("block")` util? That will also apply the `[hidden]` selector on lines 9-11 so that can be removed.

Reply:
Yes. That seems to work great! Anytime that I use the display util does that mean that I no longer need the [hidden] selector?

Reply:
Yep! You can see exactly what it's doing here: https://github.com/microsoft/fast-dna/blob/master/packages/web-components/fast-foundation/src/utilities/style/display.ts

Reply:
thank you so much. That makes it very clear :)